src/Workspaces/Core/Portable/Log/FunctionId.cs

```diff
@@ -405,7 +405,7 @@ internal enum FunctionId
 
         SnapshotService_RequestAssetAsync,
 
-        CompilationService_GetCompilationAsync,
+        // obsolete: CompilationService_GetCompilationAsync,
```

Comment:
❔ Should we keep this entry but mark it obsolete to avoid renumbering?

Reply:
I think it'd be even better to assign explicit numbers to the members.

Reply:
I wish there was a refactoring that does that :)

Reply:
Fortunately we have C# interactive...